release/rllib_tests/learning_tests/yaml_files/bc/bc-halfcheetahbulletenv-v0.yaml

```diff
@@ -6,21 +6,29 @@ bc-halfcheetahbulletenv-v0:
         # Can not check throughput for offline methods.
         # timesteps_total: 10000000
     stop:
-        time_total_s: 7200
+        time_total_s: 3600
     config:
         # Use input produced by expert SAC algo.
-        input: ["~/halfcheetah_expert_sac.zip"]
+        framework: "torch"
+        input: "dataset"
+        input_config:
+            format: "json"
+            paths: "/home/ray/halfcheetah_expert_sac.zip"
         actions_in_input_normalized: true
+        # Switch off input evaluation (data does not contain action probs).
+        off_policy_estimation_methods: {}
 
         num_gpus: 1
 
         model:
             fcnet_activation: relu
             fcnet_hiddens: [256, 256, 256]
-
-        evaluation_num_workers: 1
+        num_workers: 8
```

Comment:
this dataset is tiny, why do we need 8 workers for?

Reply:
it is but increasing the number of workers here decreases sampling time.